indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -295,6 +295,21 @@ class BookingEditForm extends React.Component {
             disabled={submitSucceeded}
           />
         </Segment>
+
+        {room.canUserViewInternalNotes && (
+          <Segment inverted styleName="booking-notes-section">
+            <p style={{marginBottom: '0.5em'}}>
+              <Translate>
+                Internal notes about the booking that will only be shown to room booking managers
```

Comment:
```suggestion
                Internal notes about the booking that will only be shown to managers
```

or "to room managers", but omitting "room" makes it work more nicely in contexts like burotel where we book desks and not rooms